### ISSUE TITLE
Clarification about account_info RPC command

### ISSUE THREAD

**Issue by Fiono11** [nanocurrency/nano-docs#501]
When the optional fields `representative`, `weight` and `pending` are `true` should the fields `confirmation_height`, `confirmation_height_frontier` and `account_version` be included in the response as well (right now they aren't)?
```
{
  "frontier": "FF84533A571D953A596EA401FD41743AC85D04F406E76FDE4408EAED50B473C5",
  "open_block": "991CF190094C00F0B68E2E5F75F6BEE95A2E0BD93CEAA4A6734DB9F19B728948",
  "representative_block": "991CF190094C00F0B68E2E5F75F6BEE95A2E0BD93CEAA4A6734DB9F19B728948",
  "balance": "235580100176034320859259343606608761791",
  "modified_timestamp": "1501793775",
  "block_count": "33",
  "confirmation_height" : "28",
  "confirmation_height_frontier" : "34C70FCA0952E29ADC7BEE6F20381466AE42BD1CFBA4B7DFFE8BD69DF95449EB",
  "account_version": "1",
  "representative": "nano_3t6k35gi95xu6tergt6p69ck76ogmitsa8mnijtpxm9fkcm736xtoncuohr3",
  "weight": "1105577030935649664609129644855132177",
  "pending": "2309370929000000000000000000000000"
}
```

**Comment by zhyatt** [comment]
@Fiono11 I tested this using a V22DB6 node as well as V21.3RC2 node and the fields `confirmation_height`, `confirmation_height_frontier` and `account_version` do appear when those other optional fields are set to `true`. Can you clarify what node version you are running and what you are seeing vs. what is expected?

**Comment by Fiono11** [comment]
I didn't try the rpc command, the point was to add those fields in the [documentation](https://docs.nano.org/commands/rpc-protocol/#account_info) when the optional fields are set to `true`.

**Comment by zhyatt** [comment]
Ah I see, good find. Fixed by 48f95e4160e536345a929e327ed61ce3d7610c01
